src/index.js

```diff
@@ -15,7 +15,8 @@ const Builder = ({
   fields = [],
   button = {},
   pagination = {},
-  logic
+  logic,
+  isSingleField = false
```

Comment:
Lets get this parameter from pagination object 'Cause it's related to pagination